src/client/sandbox/child-window/index.ts

```diff
@@ -88,10 +88,44 @@ export default class ChildWindowSandbox extends SandboxBase {
 
             e.preventDefault();
 
-            this._openUrlInNewWindow(url);
+            this._openNewWindowAfterAllEventHandlers(url, e);
```

Comment:
I propose to rename root the function as `openUrlInNewWindowIfNotPrevented`.
The internal function renames as `openUrlInNewWindowIfNotPreventedHandler`.

Reply:
ok